### PR TITLE
Add Clojure implementation of maybe-hugs

### PR DESCRIPTION
**Pull request by seancorfield**

### CODE REVIEW

**Comment by zkat** [comment]
Awesome! Thanks for this, @seancorfield.

So as far as licensing goes, I've set up the toplevel license to be public domain, using CC0-1.0. Are you okay with your contributions falling under that license, instead?

**Comment by seancorfield** [comment]
Yup, no problem. Leiningen (the Clojure build tool) just uses EPL by default because that's the license Clojure itself uses.

**Comment by seancorfield** [comment]
Do you want me to change the LICENSE / README and resubmit the PR?

**Comment by zkat** [comment]
You can just change it and push. The PR will update automatically. :) I'll merge it after that -- looks great! :D

**Comment by seancorfield** [comment]
Removed LICENSE file and License note in README so it falls under the top-level license.

File: clojure/project.clj [hunk 1]
```diff
@@ -0,0 +1,9 @@
+(defproject maybe-hugs "0.1.0-SNAPSHOT"
+  :description "A Clojure implementation of zkat's maybe-hugs"
+  :url "https://github.com/zkat/maybe-hugs/"
+  :license {:name "Eclipse Public License"
```
Comment (zkat) [inline clojure/project.clj]: Straggler here!

**Comment by seancorfield** [comment]
The straggler has been fixed! Thanks for having such sharp eyes!

**Comment by zkat** [comment]
:sheep: 

``` clojure
(hug @seancorfield)
```

**Pull request merged**